Sources/App/Controllers/SearchController.swift

```diff
@@ -30,17 +30,12 @@ struct SearchController {
                           pageSize: Constants.resultsPageSize).get()
         
         let matchedKeywords = response.results.compactMap { $0.keywordResult?.keyword }
-        
-        let weightedKeywords: [WeightedKeywordModel]
 
-        if page == 1 {
-            weightedKeywords = try await WeightedKeyword.query(on: req.db,keywords: matchedKeywords)
-                .map { WeightedKeywordModel(keyword: $0.keyword,
-                                             weight: $0.count) }
-        } else {
-            weightedKeywords = []
-        }
-        
+        // We're only displaying the keyword sidebar on the first search page.
+        let weightedKeywords = (page == 1)
```

Comment:
I specifically avoided using the tertiary statement in my old. code. But seeing it like this, yes. Agreeable code.

Reply:
Yeah, I only collapsed it because we can now get rid of the additional map and shortened the whole expressions enough to bring it sort of into `?:` territory :)